experimental/devices/mcp9808/mcp9808.go

```diff
@@ -382,14 +376,19 @@ var (
 	errTooShortInterval     = errors.New("too short interval for resolution")
 )
 
-func alertBitsToTemperature(b uint16) physic.Temperature {
-	b = (b >> 2) & 0x07FF
-	t := physic.Temperature(b&0x03FF) * 250 * physic.MilliKelvin
-	if b&0x400 > 0 {
+// bitsToTemperature converts the given bits to a physic.Temperature, assuming the
+// bit layout common to the ambient temperature register and the alert registers.
+// This works for the alert registers because while they do not make use of the 2
+// least significant bits (i.e. they have resolution of 0.25°C vs. 0.0625°C for the
+// ambient temp register) those 2 bits are always read as 0. See page 22 of the
+// datasheet.
+func bitsToTemperature(b uint16) physic.Temperature {
+	t := physic.Temperature(b&0x0fff) * 62500 * physic.MicroKelvin
+	if b&0x1000 > 0 {
```

Comment:
Since this is checking for a single bit, I prefer !=. 

In fairness I should have commented this when the code was added. Also lines 185, 187, 197 and 207.